lab-michael/test/queue-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let expect = require('chai').expect;
+let Queue = require('../lib/queue.js');
+
+describe('queue model', function() {
+  describe('.enqueue()', function() {
+    it('should throw error if there is nothing in queue', function() {
+      let q = new Queue();
+      let err = new ReferenceError('Nothing in queue');
+      let fn = function(){throw err;};
+      q.enqueue(null);
+      console.log(q);
+      expect(fn).to.throw(err);
+    });
+    it('should add an incoming connection to the end of queue', function(){
+      let q = new Queue();
+      q.enqueue(1);
+      console.log(q);
```

Comment:
Ditch the logs and comments when you submit, please. Put them on another branch if you'd like to retain them for your ongoing work.